ofl/redditsans/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2023 The Reddit Sans Project Authors (https://github.com/reddit/redditsans)
```

Comment:
If they ask for permission to keep their own copyright string in in the font file, it should be consistent here too. The ideal would be to have GF standard copyright string everywhere but maybe we can see that with an update.